backend/src/models/constants.js

```diff
@@ -356,6 +362,10 @@ exports.TABLE_COLUMNS = {
         picture: {
             type: TYPES.VarChar,
             options: { nullable: true }
+        },
+        permissions: {
```

Comment:
This should probably be a table? To enforce `enum`-ness.

Reply:
Yeah maybe but then joinszzz hnng. We have constants - are they not good enough?
We will need another table for a more complex permissions model but until then...cbf?

Reply:
okeyyy